docs/conf.py

```diff
@@ -19,7 +19,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'honeybee'
-copyright = '2019, Ladybug Tools'
+copyright = '2021, Ladybug Tools'
```

Comment:
Thank you for catching this. Mostapha actually found a great solution for this situation, which will ensure that the year is always correct, [which you can see here in honeybee-radiance](https://github.com/ladybug-tools/honeybee-radiance/blob/master/docs/conf.py#L23).  If you could implement the same here, that would be a big help.